pkg/webhook/validation/dynakube/deprecation_test.go

```diff
@@ -27,20 +27,20 @@ func TestDeprecationWarning(t *testing.T) {
 		assert.True(t, dynakube.FeatureAutomaticInjection())
 	})
 
-	t.Run(`warning present`, func(t *testing.T) {
+	t.Run(`warning not present anymore`, func(t *testing.T) {
 		dynakubeMeta := defaultDynakubeObjectMeta
 		split := strings.Split(dynatracev1beta1.AnnotationFeatureAutomaticInjection, "/")
 		postFix := split[1]
 		dynakubeMeta.Annotations = map[string]string{
-			dynatracev1beta1.DeprecatedFeatureFlagPrefix + postFix: "true",
+			`alpha.operator.dynatrace.com/feature-` + postFix: "false",
 		}
 		dynakube := &dynatracev1beta1.DynaKube{
 			ObjectMeta: dynakubeMeta,
 			Spec: dynatracev1beta1.DynaKubeSpec{
 				APIURL: testApiUrl,
 			},
 		}
-		assertAllowedResponseWithWarnings(t, 1, dynakube)
+		assertAllowedResponseWithWarnings(t, 0, dynakube)
 		assert.True(t, dynakube.FeatureAutomaticInjection())
 	})
```

Comment:
does this test make sense?

testing that something was removed feels a bit like over doing it